rust/lib/src/lib.rs

```diff
@@ -224,7 +246,6 @@ mod tests {
     #[ignore]
     #[tokio::test]
     async fn unconnected_client_framework() {
-        // Use test in RustFFITest.kt as template
```

Comment:
this test is failing and not necessary as its covered by a test in offline testsuite
please remove this whole test mod
https://github.com/zingolabs/zingo-mobile/pull/630/files#diff-4fe8b5135dd27e9e8239c61835dafa0713eea327d1f02b33f36d1ed9b17bb129R242-R268

Reply:
done! thanks.